src/gtpv1u/gtpv1u.cpp

```diff
@@ -204,7 +204,7 @@ void gtpu_l4_stack::send_g_pdu(
             reinterpret_cast<uintptr_t>(payload) -
             (uintptr_t) sizeof(struct gtpu_extn_pdu_session_container));
 
-    gtpu_ext_hdr->message_length = htobe16(1);
+    gtpu_ext_hdr->message_length = 0x01;
```

Comment:
@orion-belt Rohan

Why did you hard-code that line?

Regards
Raphael

Reply:
Length of the fields are variable. And we use only one field for now (QFI) of length one byte.